src/RequestsLibrary/log.py

```diff
@@ -42,3 +43,16 @@ def format_data_to_log_string(data, limit=LOG_CHAR_LIMIT):
         data = "%s... (set the log level to DEBUG or TRACE to see the full content)" % data[:limit]
 
     return data
+
+
+def redact_secure_header(header):
+    """Redact the secure headers to be logged."""
+    secure_headers = ('authorization', 'x-auth-token',
+                      'x-subject-token', 'x-service-token')
+    for key in header.keys():
+        if key.lower() in secure_headers:
+            token_hasher = hashlib.sha256()
+            token_hasher.update(header[key].encode('utf-8'))
+            token_hash = token_hasher.hexdigest()
+            header[key] = '{SHA256}%s' % token_hash
```

Comment:
I'm wondering whether this is the original request header structure, are you modifying the original request headers?
I would modify only the headers for logging

Reply:
>  'x-auth-token', 'x-subject-token', 'x-service-token'

These headers seem specific to how @vineet-pruthi  system-under-test. Perhaps there is a way to for users to configure headers to be redacted?

Reply:
I felt hashing auth headers information is good when just showing info level logs, and when debug level is 1, so the auth headers be shown without hashing,

Reply:
> 
> 
> I'm wondering whether this is the original request header structure, are you modifying the original request headers?
> I would modify only the headers for logging

The idea for hashed headers is to secure info if someone checks API requests test report/logs ,.. but it might confuse someone who checks the logs. i think users should get a small message as well in info logs, headers are hashed.
What's your suggestion ?